package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mojaloop/central-services-shared",
-  "version": "9.3.1-snapshot",
+  "version": "9.3.2-snapshot",
```

Comment:
hi Steven, was just checking the code coverage and we do have >90% coverage at all levels, so I think we can move this to 'release' status.. So, please remove the "-snapshot" suffix and just make this v9.3.2 and we need to reflect this in the GitHub release as well